rca/people/models.py

```diff
@@ -872,6 +880,44 @@ def student_related_links(self):
             for item in self.relatedlinks.all()
         ]
 
+    def save(self, *args, **kwargs):
+        """On saving the student page, make sure the student_user_account
+        has a group created with the necessary permissions
+        """
```

Comment:
To do - only create these permissions if:
- New self.id: the page is new
- image_collection or student_user values on the page have changed

Reply:
Potentially move this to the forms save method too https://docs.wagtail.io/en/stable/advanced_topics/customisation/page_editing_interface.html#customising-generated-forms

Reply:
Is not creating these relationships when moving this to a form down to commit=False ?